documentation/misc/zookeeper-manual-scaling-methods.md

```diff
@@ -0,0 +1,149 @@
+# Zookeeper 3.5.x Manual Scaling Procedures
+
+Zookeeper 3.5.x uses a new configuration procedure compare to 3.4.x servers. This 
+[dynamic reconfiguration](https://zookeeper.apache.org/doc/r3.5.7/zookeeperReconfig.html) means that servers need to be
+added and removed via the Zookeeper command line client (or Admin API) in order to properly maintain a stable Zookeeper 
+cluster. 
+
+The procedures below outline the steps required to scale up (add) and scale down (remove) servers from an existing 
+Strimzi Kafka deployment.
+
+## Scale up procedure
+
+The cluster should be scaled **one server at a time**. 
+
+1) Create an initial Strimzi cluster with n=3 replicas (3 Kafka + 3 ZK servers).
+
+2) Set the replica count to n=4 in the zookeeper section of the Kafka CR.
+
+3) Allow the ZK server (zookeeper-3) to start up normally and establish a link to the existing quorum.
+
+This can be checked with the following command:
+
+```
+kubectl exec -n myproject -it my-cluster-zookeeper-3 -c zookeeper -- bash -c "echo 'srvr' | nc 127.0.0.1 21813 | grep 'Mode:'"
+```
+
+Which should show something like:
+
+```
+Mode: follower
+```
+
+4) Open a zookeeper-shell session on one of the nodes in the original cluster (in this case nodes 0, 1 or 2):
+
+```
+kubectl exec -n myproject -it my-cluster-zookeeper-0 -c zookeeper -- ./bin/zookeeper-shell.sh localhost:21810
+```
+
+5) Enter the following line to add the new server to the quorum as a voting member:
+
+```
+reconfig -add server.4=127.0.0.1:28883:38883:participant;127.0.0.1:21813
```

Comment:
This needs to again stress out the differences ... and also point out that the `server.X` has different index than all the ports.